packages/fondue/src/components/RichTextEditor/Plugins/LinkPlugin/FloatingLink/CustomFloatingLink.tsx

```diff
@@ -55,21 +56,30 @@ export const CustomFloatingLink = () => {
 
     return (
         <>
-            {insertState.isOpen && !editState.isOpen && (
-                <div
-                    ref={insertRef}
-                    {...insertProps}
-                    style={{ ...insertProps.style, zIndex: zIndexLayers.floatingModal }}
-                >
-                    {input}
-                </div>
-            )}
+            {insertState.isOpen &&
+                !editState.isOpen &&
+                createPortal(
+                    <div
+                        ref={insertRef}
+                        {...insertProps}
+                        style={{ ...insertProps.style, zIndex: zIndexLayers.floatingModal }}
```

Comment:
if you are using Portal, let's leverage the DOM order layering instead of adding `zIndex`

Reply:
Fair point, adjusted 👍